lib/hello_phoenix_web/views/hello_launchdarkly_live.ex

```diff
@@ -0,0 +1,36 @@
+defmodule HelloPhoenixWeb.HelloLaunchdarklyLive do
+  use HelloPhoenixWeb, :live_view
+#  use Phoenix.LiveView
+
+  def render(assigns) do
+    case assigns.alternate do
```

Comment:
The old example had some HTML template files, but I did this inline similar to JSX.